### PR TITLE
Edit book test

### PR DESCRIPTION
**Pull request by ghost**
## Description
Testing edit book
## Reference Requirements
- Product Backlog ID: #PBI-4
- Task ID: #4.6

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: tests/Browser/Pages/Books/EditBookTest.php [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+                    ->visit('admin/books')
+                    ->resize(1600, 2000)
+                    ->clickLink(2)
+                    ->click('.btn-edit-1')
+                    ->clickLink('Back')
+                    ->assertQueryStringHas('page', 2);
+
+            $browser->visit('admin/books/1/edit')
+                    ->resize(1600, 2000)
+                    ->clickLink('Back');
```
Comment (at-annguyen) [inline tests/Browser/Pages/Books/EditBookTest.php]: test tiếp theo

File: tests/Browser/Pages/Books/EditBookTest.php [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+     */
+    public function testPressResetButton() {
+        $book = Book::findOrFail(1);
+        $category = Category::findOrFail(1);
+        $this->browse(function (Browser $browser) use ($book, $category) {
+            $browser->loginAs(User::find(1))
+                    ->visit('admin/books/1/edit')
+                    ->resize(1600, 2000)
+                    ->type('name', 'Example Book')
+                    ->select('category_id', $category->id)
```
Comment (at-annguyen) [inline tests/Browser/Pages/Books/EditBookTest.php]: giả sử thay đổi category->id vẫn test được mà

**Review by annq3012 — CHANGES_REQUESTED** [review]

**Review by annq3012 — APPROVED** [review]

**Comment by at-annguyen** [comment]
LGTM

**Comment by at-huydq** [comment]
Good job, LGTM.

**Pull request merged**